content/demo/index.html

```diff
@@ -4,10 +4,88 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>GlueTemplate - Demo</title>
+    <title>Temperature Converter - Glutinum Template</title>
+    <meta name="description" content="Simple temperature converter built with F# and Fable">
```

Comment:
I would not put an active `description` tag in the template.

Could be hinted to the user using an HTML comment